server/src/main/java/org/spine3/server/event/DispatcherEventExecutor.java

```diff
@@ -150,4 +146,19 @@ public boolean apply(@Nullable EventDispatcher input) {
             }
         };
     }
+
+    /** Utility wrapper class for predefined executors designed to be constants */
+    private static class PredefinedExecutors {
+
+        /**
+         * A pre-defined instance of the {@code DispatcherEventExecutor}, which does not postpone any event dispatching
+         * and uses {@link MoreExecutors#directExecutor()} for operation.
+         */
+        private static final DispatcherEventExecutor DIRECT_EXECUTOR = new DispatcherEventExecutor() {
```

Comment:
> `DIRECT_EXECUTOR`
This is not a constant. Have a look at Google guidelines as to naming such things.